### DISCUSSION TITLE
Image inside cell goes to next line/row

### DISCUSSION THREAD

**Discussion by shaahin4** [QuestPDF/QuestPDF#832] [Q&A] [answered]
I have table with tow columns, which each one contains an image, but second one placed on new line (row). My idea is, they should be placed side by side, in its own column.
I tried with different size of width and height, and various property like fitheight, fitwidth etc. but no seccess.
Any help please.
here is my code: 
```

 table.ColumnsDefinition(columns =>
{
    columns.ConstantColumn(200);
    columns.ConstantColumn(200); 
});
table.Cell().Element(e =>
                                {
                                    if (System.IO.File.Exists(itemCardPath))
                                    {
                                        x.Item().Width(2, Unit.Centimetre)
                                          .Image(itemCardPath)
                                          .WithRasterDpi(80); 
                                    }
                                }); 
                                table.Cell().Element(e =>
                                {
                                    if (System.IO.File.Exists(itemCardPath2))
                                    {
                                        x.Item().Width(2, Unit.Centimetre)
                                          .Image(itemCardPath).FitWidth();
                                    }
                                });
```

**Comment by girlpunk** [comment]
In your sample code, what is `x`?

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Unfortunately, as also noted by another community member, the provided code example contains undefined references (e.g. `x`), which makes it difficult for us to reproduce the issue and provide a working solution.

In the meantime, the following documentation pages may help with your table layout:
- [Cell Style Pattern](https://www.questpdf.com/api-reference/table/cell-style-pattern.html)
- [Overlapping Cells](https://www.questpdf.com/api-reference/table/overlapping-cells.html)

Should you encounter any further problems, please don't hesitate to open a new issue with a complete, minimal code example so we can assist you more effectively.

Thank you!
